http/headers/clear-site-data.json

```
@@ -63,11 +63,28 @@
               "edge": {
                 "version_added": "≤79"
               },
-              "firefox": {
-                "version_added": "63"
-              },
+              "firefox": [
+                {
+                  "version_added": "94",
+                  "partial_implementation": true,
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "#privacy.clearsitedata.cache.enabled",
```

Firefox doesn't use the hash notation

```suggestion
                      "name": "privacy.clearsitedata.cache.enabled",
```